travelcheck/prices.py

```diff
@@ -113,15 +128,24 @@ def index(self):
 
             if not result:
                 logging.info("Adding subscription")
-                result = kiwi.subscribe(subscription)
+                result = kiwi.subscribe(subscription, self._config_kiwi)
                 result.update(subscription)
                 self._db.add_subscription(result)
+            else:
+                logging.info(
+                    "Found subscription: %s" % json.dumps(result, indent=4, default=json_util.default))
 
-            response = Prices.__get_response(result, deeplink_type)
+            response = Prices.__make_response(result)
             return response
 
+        # Return the following in case of a failed request to Kiwi
+        # To do: return the search deeplink instead of none
+
         except Exception as err:
-            logging.error("Error: %s" % err)
+            
+            #logging.error("Error: %s" % err)
```

Comment:
what's the point of having this line commented?

Reply:
I replaced it with: 
logging.exception(err)
-- returns more details (for example, line number)